[2022-06-07 21:53:02,663] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: CSV_TO_DS.create_schema_and_tables_logs manual__2022-06-07T21:53:00.835001+00:00 [queued]>
[2022-06-07 21:53:02,671] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: CSV_TO_DS.create_schema_and_tables_logs manual__2022-06-07T21:53:00.835001+00:00 [queued]>
[2022-06-07 21:53:02,671] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-07 21:53:02,671] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-07 21:53:02,671] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-07 21:53:02,683] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_schema_and_tables_logs> on 2022-06-07 21:53:00.835001+00:00
[2022-06-07 21:53:02,687] {standard_task_runner.py:52} INFO - Started process 3667 to run task
[2022-06-07 21:53:02,690] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'CSV_TO_DS', 'create_schema_and_tables_logs', 'manual__2022-06-07T21:53:00.835001+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/csv_to_ds.py', '--cfg-path', '/tmp/tmprvzj4qwn', '--error-file', '/tmp/tmpdm8ad1lp']
[2022-06-07 21:53:02,691] {standard_task_runner.py:80} INFO - Job 108: Subtask create_schema_and_tables_logs
[2022-06-07 21:53:02,752] {task_command.py:370} INFO - Running <TaskInstance: CSV_TO_DS.create_schema_and_tables_logs manual__2022-06-07T21:53:00.835001+00:00 [running]> on host 33a1fdf44d41
[2022-06-07 21:53:02,836] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=CSV_TO_DS
AIRFLOW_CTX_TASK_ID=create_schema_and_tables_logs
AIRFLOW_CTX_EXECUTION_DATE=2022-06-07T21:53:00.835001+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-07T21:53:00.835001+00:00
[2022-06-07 21:53:02,847] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-06-07 21:53:02,851] {dbapi.py:213} INFO - Running statement: CREATE SCHEMA IF NOT EXISTS logs;

CREATE TABLE IF NOT EXISTS logs.load_csv_to_ds (
    ID SERIAL PRIMARY KEY,
    TABLE_NAME INT,
    CSV_PATH INT,
    TIME_START_LOAD TIMESTAMP DEFAULT CURRENT_TIMESTAMP(0),
    TIME_END_LOAD TIMESTAMP,
    condition TEXT
);, parameters: None
[2022-06-07 21:53:02,852] {postgres.py:94} INFO - NOTICE:  schema "logs" already exists, skipping

[2022-06-07 21:53:02,852] {postgres.py:94} INFO - NOTICE:  relation "load_csv_to_ds" already exists, skipping

[2022-06-07 21:53:02,865] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=CSV_TO_DS, task_id=create_schema_and_tables_logs, execution_date=20220607T215300, start_date=20220607T215302, end_date=20220607T215302
[2022-06-07 21:53:02,909] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-07 21:53:02,950] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
